src/node_crypto.cc

```diff
@@ -4957,8 +4956,12 @@ void ECDH::ComputeSecret(const FunctionCallbackInfo<Value>& args) {
 
   EC_POINT* pub = ecdh->BufferToPoint(Buffer::Data(args[0]),
                                       Buffer::Length(args[0]));
-  if (pub == nullptr)
+  if (pub == nullptr) {
+    args.GetReturnValue().Set(
+      String::NewFromUtf8(env->isolate(),
+      "ERR_CRYPTO_ECDH_KEY_FAIL_TRANSLATE"));
```

Comment:
Style nit: can you indent line continuations by four spaces?

Since you're passing a static string, you can use the slightly more efficient`FIXED_ONE_BYTE_STRING(env->isolate(), "...")` here.

Reply:
thanks!
this also helped me to correct the error code in c++(which was one of the first names i thought but then dropped for `ERR_CRYPTO_ECDH_INVALID_PUBLIC_KEY`) 👍